__tests__/app.test.js

```diff
@@ -224,3 +224,94 @@ describe("POST /api/reviews/:review_id/comments", () => {
       });
   });
 });
+
+describe("PATCH /api/reviews/:review_id", () => {
+  test("200: responds with updated review object with the correctly incremented votes property", () => {
+    return request(app)
+      .patch("/api/reviews/1")
+      .send({ inc_votes: 1 })
+      .expect(200)
+      .then((response) => {
+        const { updatedReview } = response.body;
+        expect(updatedReview).toMatchObject({
+          review_id: 1,
+          title: expect.any(String),
+          review_body: expect.any(String),
+          designer: expect.any(String),
+          review_img_url: expect.any(String),
+          votes: 2,
+          category: expect.any(String),
+          owner: expect.any(String),
+          created_at: expect.any(String),
+        });
+      });
+  });
+  test("200: responds with updated review object with the correctly decremented votes property", () => {
+    return request(app)
+      .patch("/api/reviews/5")
+      .send({ inc_votes: -5 })
+      .expect(200)
+      .then((response) => {
+        const { updatedReview } = response.body;
+        expect(updatedReview).toMatchObject({
+          review_id: 5,
+          title: expect.any(String),
+          review_body: expect.any(String),
+          designer: expect.any(String),
+          review_img_url: expect.any(String),
+          votes: 0,
+          category: expect.any(String),
+          owner: expect.any(String),
+          created_at: expect.any(String),
+        });
+      });
+  });
+  test("200: responds with original review with the votes unaffected, when sent an empty request", () => {
+    return request(app)
+      .patch("/api/reviews/3")
+      .send({})
```

Comment:
If the sent body does not contain inc_votes this should be a bad request error as this is not the desired input.